lib/rubocop/cop/rspec/predicate_matcher.rb

```diff
@@ -277,7 +277,7 @@ def replacement_matcher(node)
       #   expect(foo).to be_something
```

Comment:
Actually this one should be updated as well

Reply:
To what exactly?

Reply:
Oh, sorry, misread the examples :facepalm: 
Seems not a good day for me to do reviews :)

Reply:
What about line 285?

Reply:
Good catch, missed that other one!